### PR TITLE
Release v0.8.7

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/android/app/build.gradle [hunk 1]
```diff
@@ -121,8 +121,8 @@ android {
         applicationId "com.apolloschurch.app"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 11
-        versionName "0.8.6"
+        versionCode 12
```
Comment (IsaacHardy) [inline packages/apolloschurchapp/android/app/build.gradle]: Why is this needed to be incremented upon each NPM release?

Comment (vinnyjth) [reply packages/apolloschurchapp/android/app/build.gradle]: Our package that updates the `versionName` also happens to update the versionCode. A little weird, but also not an issue because `AppCenter` tracks it's own `versionCode`

**Comment by IsaacHardy** [comment]
LGTM
